server/main_chatbot.py

```
@@ -296,6 +296,72 @@ def save_incident_thought(content: str, force: bool = False):
+                            "text": accumulated_text,
+                            "_streaming": True,
+                        })
+
+                    cursor.execute(
+                        "UPDATE chat_sessions SET messages = %s, updated_at = %s WHERE id = %s",
+                        (json.dumps(messages), datetime.now(), session_id),
+                    )
+                conn.commit()
+                last_chat_save_time[0] = current_time
+                logger.debug(f"[BackgroundChat] Saved streaming message for session {session_id}: {len(accumulated_text)} chars")
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)